### ISSUE TITLE
module for custom nodes: cannot import name 'AutoencoderKLCogVideoX' from 'diffusers.models

### ISSUE THREAD

**Issue by pondloso** [kijai/ComfyUI-CogVideoXWrapper#4]
Cannot import E:\ComfyUI_windows_portable_nvidia\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-CogVideoXWrapper module for custom nodes: cannot import name 'AutoencoderKLCogVideoX' from 'diffusers.models' (E:\ComfyUI_windows_portable_nvidia\ComfyUI_windows_portable\python_embeded\Lib\site-packages\diffusers\models\__init__.py)

Look like  diffusers in portable in cant use CogVideo yet.
Is there a way to upgrade diffusers   in the portable comfyui form this "huggingface_hub
git+https://github.com/huggingface/diffusers.git@878f609aa5ce4a78fea0f048726889debde1d7e8#egg=diffusers"?

**Comment by kijai** [comment]
Yes, you just have to use the portable python.exe in the `ComfyUI_windows_portable\python_embeded` -folder:

`python.exe -m pip install it+https://github.com/huggingface/diffusers.git@878f609aa5ce4a78fea0f048726889debde1d7e8#egg=diffusers`

**Comment by pondloso** [comment]
Thank you for your work ,every thing work fine now.
but look like vid2vid is not work ,out put video not like input at all .
if you reduce denoise it will just give out blur video.

**Comment by kijai** [comment]
> Thank you for your work ,every thing work fine now. but look like vid2vid is not work ,out put video not like input at all . if you reduce denoise it will just give out blur video.

It depends a lot on the input video, sometimes it works well and sometimes it's just noisy trash. It's very experimental, it wasn't official feature at all, just there to experiment with.
